src/Razor/src/Microsoft.VisualStudio.LanguageServerClient.Razor/DefaultRazorLanguageServerCustomMessageTarget.cs

```diff
@@ -41,7 +40,9 @@ public class DefaultRazorLanguageServerCustomMessageTarget : RazorLanguageServer
 
             if (_documentManager is null)
             {
+#pragma warning disable CA2208 // Instantiate argument exceptions correctly
                 throw new ArgumentNullException(nameof(_documentManager));
```

Comment:
Can probably just throw an `ArgumentException(nameof(documentManager))` here to avoid this. Don't feel strongly though.

Reply:
`ArgumentException`s first argument is a message though isn't it?